pkg/oc/lib/describe/deployments_test.go

```diff
@@ -61,43 +62,44 @@ func TestDeploymentConfigDescriber(t *testing.T) {
 		return output
 	}
 
-	podList.Items = []kapi.Pod{*mkPod(kapi.PodRunning, 0)}
-	out := describe()
-	substr := "Autoscaling:\tbetween 1 and 3 replicas"
-	if !strings.Contains(out, substr) {
-		t.Fatalf("expected %q in output:\n%s", substr, out)
-	}
+	podList.Items = []corev1.Pod{*mkV1Pod(corev1.PodRunning, 0)}
+	/*
```

Comment:
make a TODO here as well to be sure

Reply:
might worth an issue to re-enable the test

Reply:
https://github.com/openshift/origin/issues/20394